content/en/docs/concepts/windows/intro.md

```diff
@@ -353,6 +353,24 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Worker Node Hardware Recommendations {#windows-node-hardware-recommendations}
+
+The minimum hardware requirements for a Windows worker node will depend on multiple factors, such as the Windows operating system version, and also your workload requirements.
```

Comment:
Could you please wrap these long lines appropriately as we do for other paragraphs?

This is still marked as a WIP ... are you still working on it?

Reply:
Yes I am still working on it, I will finalize the PR and open it soon.